guides/v2.3/rest/bulk-endpoints.md

```diff
@@ -7,10 +7,10 @@ functional_areas:
   - Integration
 ---
 
-Bulk API endpoints differ from other endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
+Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue. 
 
 {:.bs-callout .bs-callout-tip}
-GET requests are not supported.
+Use the `bin/magento queue:consumer:start async.operations.all` command to enable bulk endpoint processing.
```

Comment:
I would recommend to use 
queue:consumers:start

queue:consumer:start -> will also work correctly, but if you will do just bin/magento there will be no such command listed. This may cause some misunderstanding.